prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,49 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
```

Comment:
Err, why would org membership ever prevent you from unassigning someone? They shouldn't be assigned in the first place!